File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.cs
using System;

public class Solution 
{
    public string solution(string rsp) 
    {
        string rspCounter = "";
        for(int i=0; i<rsp.Length; i++) 
        {
            switch(rsp[i].ToString())
            {         
                case "2":
                    rspCounter += "0";
                    break;
                case "0":
                    rspCounter += "5";
                    break;
                case "5":
                    rspCounter += "2";
                    break;
            }
        }
        return rspCounter;
    }
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string rsp) 
{
    string result = "";
    
    for(char &c : rsp)
    {
        if(c == '2')
        {
            result += '0';
        }
        else if(c == '0')
        {
            result += '5';
        }
        else
        {
            result += '2';
        }
    }
    return result;
}